applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc/MaterialParameters1.json

```diff
@@ -23,16 +23,10 @@
           "POISSON_RATIO": 0.2,
           "THICKNESS": 1.0,
           "K0_MAIN_DIRECTION": 1,
-          "K0_VALUE_XX": 0.5,
-          "K0_VALUE_YY": 0.5,
-          "K0_VALUE_ZZ": 0.5,
-	  "K0_NC": 0.6,
+          "K0_NC": 0.6,
           "RETENTION_LAW": "SaturatedLaw",
           "RESIDUAL_SATURATION": 0.06203,
```

Comment:
If I'm not mistaken, `"SaturatedLaw"` doesn't use `"RESIDUAL_SATURATION"`. If that is correct, can we remove that input parameter here?

Reply:
Also removed the redundant MINIMUM_RELATIVE_PERMEABILITY.